internal/app/machined/pkg/controllers/block/mount.go

```diff
@@ -548,20 +564,20 @@ func (ctrl *MountController) handleOverlayMountOperation(
 		return fmt.Errorf("overlay mount is not supported for %q", volumeStatus.TypedSpec().ParentID)
 	}
 
-	mountpoint := mount.NewVarOverlay(
+	manager := mount.NewVarOverlay(
 		[]string{mountTarget},
 		mountTarget,
-		mount.WithFlags(unix.MS_I_VERSION),
```

Comment:
`MS_I_VERSION` is no longer a valid flag. It was replaced by `iversion` fsconfig parameter, which is accepted by configs of:
- `ext4` (default enabled)
- `xfs` (always enabled)
- `btrfs`
- `f2fs`